extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -179,7 +234,7 @@ public function buildRemovePackage ( $oPage, $oUser, $page_id ) {
 		return $logid;
 	}
 
-	public function buildUndeletePackage( $oTitle, $created = false ) {
+	public function buildUndeletePackage( $oTitle ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)